lib/util/cache/package/redis.ts

```diff
@@ -1,5 +1,5 @@
 /* istanbul ignore file */
-import { IHandyRedis, createHandyClient } from 'handy-redis';
+import { IHandyRedis, createNodeRedisClient } from 'handy-redis';
```

Comment:
Minor - `IHandyRedis` is now a deprecated alias of `WrappedNodeRedisClient`. It won't disappear until the next major at least but may as well update it now.

Reply:
Thanks, fixed